types.go

```diff
@@ -1,12 +1,13 @@
 package main
 
 type BenchmarkConfiguration struct {
-	Benchtime string  `yaml:"benchtime"`
-	Threshold float64 `yaml:"threshold"`
-	Compare   string  `yaml:"compare"`
-	Cpu       string  `yaml:"cpu"`
-	Timeout   string  `yaml:"timeout"`
-	Benchmem  *bool   `yaml:"benchmem,omitempty"`
+	Benchtime          string  `yaml:"benchtime"`
+	Threshold          float64 `yaml:"threshold"`
+	Compare            string  `yaml:"compare"`
+	Cpu                string  `yaml:"cpu"`
+	Timeout            string  `yaml:"timeout"`
+	Benchmem           *bool   `yaml:"benchmem,omitempty"`
+	VersionRequirement string  `yaml:"version_requirement"`
```

Comment:
```suggestion
VersionRequirement string  `yaml:"versionRequirement"`
```

I think you may want to have this in the `Benchmark` struct directly because it's not applicable to `BenchmarkList`?

Reply:
Done.